programs/local/LocalServer.cpp

```diff
@@ -190,6 +200,93 @@ static void attachSystemTables(ContextPtr context)
     attachSystemTablesLocal(*system_database);
 }
 
+static void showClientVersion()
+{
+    std::cout << DBMS_NAME << " client version " << VERSION_STRING << VERSION_OFFICIAL << "." << '\n';
+}
+
+static void clearTerminal()
```

Comment:
How about moving this into some separate place? (since it is copy-paste from the client AFAICS).
Maybe something like `Common/Interactive` or `Common/InteractiveUtils`